### ISSUE TITLE
DNS Spoofing not doing anything

### ISSUE THREAD

**Issue by jigsaw678** [Trackbool/DerpNSpoof#6]
`|    \  ___  ___  ___ |   | ||   __| ___  ___  ___ |  _|
|  |  || -_||  _|| . || | | ||__   || . || . || . ||  _| 
|____/ |___||_|  |  _||_|___||_____||  _||___||___||_| 
                 |_|                |_|                 
	 Coded by Adrián Fernández Arnal-(@adrianfa5)

     --------------------------------------
    [!] Options to use:
    	  <ip>  - Spoof the DNS query packets of a certain IP address
    	  <all> - Spoof the DNS query packets of all hosts
    [!] Examples:
    	  # python3 DerpNSpoof.py 192.168.1.20 myfile.txt
    	  # python3 DerpNSpoof.py all myfile.txt
     --------------------------------------
    [i] Spoofing DNS responses...

`

Its not doing anything else

**Comment by testhaha1234** [comment]
you need to run it with arpspoof, because this tool is waiting for dns request and modify the dns response to your server, and what arpspoof is, it is that the hacker computer will pretend it to be the gateway and spoof the gateway it is the victim computer, and once it is finish the connection will become malicious

Before the arpspoof
computer --> gateway -> server

After the arpspoof
computer -> hacker -> gateway -> server

and then the hacker can now view all of your packet, and what this tool is doing, it sniff the packet and analysis 
it if it is a domain name server(DNS) request, if yes, and check the file you write into, to see where is the ip address you want to pretend it to be, and send back the malicious address instead of the real one

But dnsspoof will not working under HTTPS

which mean you can dnsspoof google facebook, see here of more https://stackoverflow.com/questions/60727879/dns-spoofing-with-python-and-netfilterqueue/65810186#65810186

**Comment by testhaha1234** [comment]
use another tool else

**Comment by testhaha1234** [comment]
Yes it is absolutely working, you might probably facing one of the issue below

1, you are trying to dnsspoof HTTPS

dnsspoof cant spoof https, cause there is something call ssl
HERE-> https://stackoverflow.com/questions/60727879/dns-spoofing-with-python-and-netfilterqueue/65810186#65810186

2, your dns is not flush
if you are in window, go try ipconfig /dnsflush

3, you didn't run it with arpspoof 

this tool can only modify the packet into the address you like and not sniff the packet
